### DISCUSSION TITLE
Will there be a guide to modify specific game settings in the near future?

### DISCUSSION THREAD

**Discussion by wang273535** [BAAI-Agents/Cradle#65] [General]
Hello! In the future, will it be possible to formulate some rules based on the operation of game players and add new game settings? Will there be a guide to modify specific game settings in the near future?

**Comment by WeihaoTan** [comment]
Hi. Thanks for reaching out. We provide an overall perspective on how to adapt the framework to a new game: https://github.com/BAAI-Agents/Cradle/discussions/55  . 

And we are still trying to re-organize the code to make it easier to understand and adapt to new game.
